metrics.go

```diff
@@ -32,23 +33,31 @@ type ConsumerMetrics struct {
 	numWorkerManagersGauge metrics.Gauge
 	activeWorkersCounter   metrics.Counter
 	pendingWMsTasksCounter metrics.Counter
+	taskTimeoutCounter     metrics.Counter
 	wmsBatchDurationTimer  metrics.Timer
 	wmsIdleTimer           metrics.Timer
 }
 
-func newConsumerMetrics(consumerName string) *ConsumerMetrics {
+func newConsumerMetrics(consumerName, prefix string) *ConsumerMetrics {
 	kafkaMetrics := &ConsumerMetrics{
-		registry: metrics.NewRegistry(),
+		registry: metrics.DefaultRegistry,
```

Comment:
Everything looks good to me except this line. I'm not sure it's a good idea to change back to DefaultRegistry. When we close the consumer we unregister all metrics in the registry and if we use default registry there's a non-zero possibility we will unregister metrics unrelated to go kafka client. Can you explain this change? Maybe I just missed something.